src/components/ex-table/ex-table.scss

```diff
@@ -0,0 +1,24 @@
+table {
+  @apply mt-component-spacing-small inline-block max-w-full overflow-x-scroll;
```

Comment:
Use overflow-x-auto. Because this way I get grayed out non-usable scroll bars:

![image](https://user-images.githubusercontent.com/88790097/135103763-c5bc6a3a-0130-48d7-bc2b-cd0ccbb97e46.png)

Reply:
used overflow-x-auto